src/operations/blas2/spr.hpp

```diff
@@ -101,20 +101,21 @@ typename rhs_1_t::value_t Spr<Single, isUpper, lhs_t, rhs_1_t, rhs_2_t>::eval(
 
   index_t row = 0, col = 0;
 
-#ifndef __ADAPTIVECPP__
```

Comment:
Why is it safe to remove the `__ADAPTIVECPP__` macros?

Reply:
Thanks for your reviews.

I rechecked and contrary to what I thought this could be still necessary. I reintroduced it in 381afab.